.gitlab-ci.yml

```diff
@@ -63,6 +63,72 @@ testing:
       - tests/JU_*.xml
       - screenshot/*.png
 
+testing9_x64:
+  stage: build
+  tags:
+    - debian, docker
+  parallel:
+    matrix:
+      - IGOR_VERSION:
+          - 9
+        EXPERIMENT:
+          - VeryTinyTestEnvironment.pxp
+          - Various.pxp
+          - IMTest.pxp
+  variables:
+    DOCKER_AUTH_CONFIG: "${DOCKER_AUTH_CONFIG_IGOR_PRO}"
+  image:
+    name: ${CI_REGISTRY}/internal/docker-igorpro:v${IGOR_VERSION}
+  script:
+    - ln -rs procedures "/home/igor/WaveMetrics/Igor Pro ${IGOR_VERSION} User Files/User Procedures/utf"
+    - ln -rs tests "/home/igor/WaveMetrics/Igor Pro ${IGOR_VERSION} User Files/User Procedures/tests"
+    - mkdir -p "/home/igor/WaveMetrics/Igor Pro ${IGORPRO_VERSION} User Files/Igor Extensions (64-bit)"
+    - ln -rs xop/*.xop "/home/igor/WaveMetrics/Igor Pro ${IGORPRO_VERSION} User Files/Igor Extensions (64-bit)"
+    - touch tests/DO_AUTORUN.TXT
+    - igorpro execute --verbose --screenshot "tests/${EXPERIMENT}"
+    - exit $(grep -e "errors=\"[0-9]\+\"" -e "failures=\"[0-9]\+\"" tests/JU_*.xml | grep -cv "failures=\"0\" errors=\"0\"")
+  artifacts:
+    when: always
+    reports:
+      junit: tests/JU_*.xml
+    paths:
+      - tests/*.log
+      - tests/JU_*.xml
+      - screenshot/*.png
+
+testing9_x86:
+  stage: build
+  tags:
+    - debian, docker
+  parallel:
+    matrix:
+      - IGOR_VERSION:
+          - 9
+        EXPERIMENT:
+          - VeryTinyTestEnvironment.pxp
+          - Various.pxp
+          - IMTest.pxp
+  variables:
+    DOCKER_AUTH_CONFIG: "${DOCKER_AUTH_CONFIG_IGOR_PRO}"
+  image:
+    name: ${CI_REGISTRY}/internal/docker-igorpro:v${IGOR_VERSION}
+  script:
+    - ln -rs procedures "/home/igor/WaveMetrics/Igor Pro ${IGOR_VERSION} User Files/User Procedures/utf"
+    - ln -rs tests "/home/igor/WaveMetrics/Igor Pro ${IGOR_VERSION} User Files/User Procedures/tests"
+    - mkdir -p "/home/igor/WaveMetrics/Igor Pro ${IGORPRO_VERSION} User Files/Igor Extensions"
+    - ln -rs xop/*.xop "/home/igor/WaveMetrics/Igor Pro ${IGORPRO_VERSION} User Files/Igor Extensions"
+    - touch tests/DO_AUTORUN.TXT
+    - igorpro execute --32bit --verbose --screenshot "tests/${EXPERIMENT}"
+    - exit $(grep -e "errors=\"[0-9]\+\"" -e "failures=\"[0-9]\+\"" tests/JU_*.xml | grep -cv "failures=\"0\" errors=\"0\"")
+  artifacts:
+    when: always
+    reports:
+      junit: tests/JU_*.xml
+    paths:
+      - tests/*.log
+      - tests/JU_*.xml
+      - screenshot/*.png
+
```

Comment:
Can we integrate that in the testing job? Like in https://github.com/AllenInstitute/ZeroMQ-XOP/blob/4c5928d22928f51f30fdcc24e600c59592e7cd5b/.gitlab-ci.yml#L50